pootle/apps/accounts/proxy.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from hashlib import md5
+
+from django.core.urlresolvers import reverse
+from django.utils.html import format_html
+
+
+class DisplayUser(object):
```

Comment:
If this is the same as `DisplayUser` in `pootle/apps/pootle_store/unit/timeline.py` perhaps commit message is better worded like "Move DisplayUser to accounts.proxy".